doc/source/contributing_docs.rst

```diff
@@ -231,23 +341,58 @@ Then, run the following command in the ``doc/`` directory:
 This will start a local webserver at http://localhost:8000/, which will refresh every time you save changes to a file in the documentation. This is helpful for both the user guide (first navigate to ``UserGuide/doc``) and the main repository documentation (navigate to ``package/doc/sphinx``).
 
 
-Using pre-commit hooks
-====================================
+Advanced preview with gh-pages
+""""""""""""""""""""""""""""""
 
-Manually editing files can often lead to small inconsistencies: a whitespace here, a missing carriage return there. A tool called pre-commit can be used to automatically fix these problems, before a git commit is made. To enable the pre-commit hooks, run the following:
+.. note::
 
-    .. code-block:: bash
+    This section documents how to render documentation on a fork without ReadTheDocs.
+    This is *generally unnecessary* and should only be done in cases where we
+    believe ReadTheDocs is not rendering our documentation properly.
+    For all other cases, please use the ReadTheDocs preview in pull requests.
 
-        pre-commit install
+It is often difficult to review Jupyter notebooks on GitHub, especially if you embed widgets and images.
```

Comment:
Should we be signposting to reviewnb here?

Reply:
Added a link